code/game/objects/structures/morgue.dm

```diff
@@ -52,6 +52,9 @@
 /obj/structure/morgue/attack_hand(mob/user)
 	toggle_morgue(user)
 
+/obj/structure/morgue/attack_alien(mob/living/carbon/Xenomorph/M)
+	toggle_morgue(M)
```

Comment:
Should most likely be a do_after so they're actually prying it open.

Reply:
No, because marines have been using this to hide from xenos; if they take time to pry open, and a marine can close it in an instant, the problem is obvious.

On the flipside, if we make the xenos just break a tray and make it lodge open forever, it can trap the xeno if it has no acid.

The only other option I see is the xeno can pry the morgue tray open and break it such that the tray is severed, and doesn't impede movement.